src/examples/accordions/stepper/StepperLayout.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Stepper } from '@zendeskgarden/react-accordions';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledParagraph = styled.div`
+  margin: 0 0 ${p => p.theme.space.sm} 0;
+`;
+
+const StyledContainer = styled.div`
+  margin: ${p => p.theme.space.md} 0 0 0;
+`;
+
+const Example = () => {
+  const [currentStep, setStep] = useState(0);
+
+  const onNext = () => setStep(currentStep + 1);
+  const onBack = () => setStep(currentStep - 1);
+
+  const allSteps = [
+    {
+      content: `Your garden's success depends on its location, so choose a spot that has
+      healthy soil, gets good light, and is easily watered.`,
+      buttons: <Button onClick={onNext}>Next</Button>
+    },
+    {
+      content: `The layout of your garden depends on its purpose. If you're planting flowers,
+      consider aesthetics like color and layout. If you're growing food, think about
+      harvest times and the kinds of pests that might be attracted to your crops.`,
+      buttons: (
+        <>
+          <Button onClick={onBack} style={{ marginRight: '12px' }}>
+            Back
+          </Button>
+          <Button onClick={onNext}>Next</Button>
+        </>
+      )
+    },
+    {
+      content: `Buy clean, hearty, disease-free seeds. Most seed from reliable seed companies will meet these specifications.`,
+      buttons: <Button onClick={onBack}>Back</Button>
+    }
+  ];
+
+  return (
+    <Row justifyContent="center">
+      <Col sm={10} textAlign="center">
+        <Stepper activeIndex={currentStep} isHorizontal>
+          <Stepper.Step key="step-1">
+            <Stepper.Label>Choose a good location</Stepper.Label>
+          </Stepper.Step>
+          <Stepper.Step key="step-2">
+            <Stepper.Label>Plan your garden&apos;s layout</Stepper.Label>
+          </Stepper.Step>
+          <Stepper.Step key="step-3">
+            <Stepper.Label>Buy great seeds</Stepper.Label>
+          </Stepper.Step>
+        </Stepper>
+        {allSteps.map(
+          (step, index) =>
+            index === currentStep && (
+              <StyledContainer>
```

Comment:
These mapped elements will need keys to help React identify which items have changed, are added, or are removed. The warnings in the console should go away once a unique key is added.